src/er.h

```diff
@@ -42,6 +47,33 @@ int ER_Create_Scheme(
   int erasure_blocks          /**< [IN] - number of erasure blocks to be generated */
 );
 
+/* needs to be above doxygen comment to get association right */
+typedef struct kvtree_struct kvtree;
+/**
+ * Get/set ER configuration values.
+ *
+ * The following configuration options can be set (type in parenthesis):
+ *   * "DEBUG" (int) - if non-zero, output debug information from inside
+ *     ER.
+ *   * "SETSIZE" (int) - set size for ER to use.
+ *   * "MPI_BUF_SIZE" (byte count [IN], int [OUT]) - MPI buffer size to chunk
+ *     file transfer. Must not exceed INT_MAX.
+ *   .
+ * Symbolic names ER_KEY_CONFIG_FOO are defined in er.h and should
+ * be used instead of the strings whenever possible to guard against typos in
+ * strings.
+ *
+ * \result If config != NULL, then return config on success.  If config=NULL
+ *         (you're querying the config) then return a new kvtree on success.
+ *         NULL on any failures.
+ * \param config The new configuration. If config=NULL, then return a kvtree
+ *                with all the configuration values.
```

Comment:
Let's note that the caller is responsible for deleting any returned kvtree.

Reply:
Added in 91b7c1a